pkg/stdlib/packages_test.go

```diff
@@ -36,8 +36,8 @@ func TestGetPackages(t *testing.T) {
 
 	builder := &build.Instance{}
 	builder.AddFile("-", `
-import "vela/op"
-out: op.context`)
+import "vela/custom"
+out: custom.context`)
```

Comment:
does it mean the API changed?

Reply:
Actually I don't think we're using this feature right now.